### PR TITLE
[TASK] Update code snippet for LanguageService class

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/Localization/LocalizationApi/_LanguageService.rst.txt [hunk 1]
```diff
@@ -41,15 +45,22 @@
       Main and most often used method.
       
       Resolve strings like these:
-           'LLL:EXT:core/Resources/Private/Language/locallang_core.xlf:labels.depth_0'
-      This looks up the given .xlf file path in the 'core' extension for label 'labels.depth_0'
+      
+      ..  code-block:: php
+          
+               'LLL:EXT:core/Resources/Private/Language/locallang_core.xlf:labels.depth_0'
```
Comment (linawolf) [inline Documentation/ApiOverview/Localization/LocalizationApi/_LanguageService.rst.txt:51]: Something is wrong with the indentation here

File: Documentation/ApiOverview/Localization/LocalizationApi/_LanguageService.rst.txt [hunk 2]
```diff
@@ -41,15 +45,22 @@
...
+      This looks up the given .xlf file path in the 'core' extension for label labels.depth_0
       
       :param string $input: Label key/reference
       :returntype: string
       
    .. php:method:: includeLLFile(string $fileRef)
    
       Includes locallang file (and possibly additional localized version if configured for)
-      Read language labels will be merged with $LOCAL_LANG (if $setGlobal = TRUE).
+      Read language labels will be merged with $LOCAL_LANG (if $setGlobal === true).
```
Comment (linawolf) [inline Documentation/ApiOverview/Localization/LocalizationApi/_LanguageService.rst.txt:62]: Comparing with true does not really make sense, does it? Maybe turn it into a sentence?

**Comment by brotkrueml** [comment]
PR for improving code blocks: https://review.typo3.org/c/Packages/TYPO3.CMS/+/77367

**Comment by brotkrueml** [comment]
The PR was merged, but the codesnippets tool does not render the new code block. This has to be investigated first (and then generated the API again with the full content).

**Comment by brotkrueml** [comment]
Too long opened, here will nothing happens anymore.

**Pull request closed without merging**